frontend/src/pages/org/dashboard.ts

```diff
@@ -78,9 +87,12 @@ export class Dashboard extends LiteElement {
     });
 
     return html`<span title="${fullFormatter.format(minutes)}">
-        ${compactFormatter.format(minutes)}</span
-      >
-      (${humanizeMilliseconds(seconds * 1000)})`;
+        ${compactFormatter.format(minutes)}
+      </span>
+      ${when(
+        humanize,
+        () => html` (${humanizeMilliseconds(seconds * 1000)})`
+      )} `;
```

Comment:
This all got refactored in https://github.com/webrecorder/browsertrix-cloud/pull/1386 — would be worth refactoring this to use & improve those utils. Specifically, `humanizeExecutionSeconds` with `style` `"short"` would do what you're looking for here, I think! Happy to help there, if you like.

Would be happy to do the rename of 'humanize' -> 'format' though, if you prefer!

Reply:
Now rebased! Thanks for the tip on `style` `"short"` :) Just working through some of the other issues now and then will mark for re-review!